rpcs3/Emu/Cell/SPULLVMRecompiler.cpp

```diff
@@ -3991,7 +3991,7 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 
 				bool must_use_cpp_functions = !!g_cfg.core.spu_accurate_dma;
 
-				if (u64 cmdh = ci->getZExtValue() & ~(MFC_BARRIER_MASK | MFC_FENCE_MASK | MFC_RESULT_MASK); g_cfg.core.rsx_fifo_accuracy || g_cfg.video.strict_rendering_mode || !g_use_rtm)
+				if (u64 cmdh = ci->getZExtValue() & ~(MFC_BARRIER_MASK | MFC_FENCE_MASK | MFC_RESULT_MASK); g_cfg.core.rsx_fifo_accuracy || g_cfg.video.strict_rendering_mode || /*g_use_rtm*/ true)
```

Comment:
Maybe keep the original in the comment instead
```suggestion
				if (u64 cmdh = ci->getZExtValue() & ~(MFC_BARRIER_MASK | MFC_FENCE_MASK | MFC_RESULT_MASK); g_cfg.core.rsx_fifo_accuracy || g_cfg.video.strict_rendering_mode || /*!g_use_rtm*/ true)
```

Reply:
might as well put it to the front